lib/features/steps/compound_steps.rb

```diff
@@ -0,0 +1,31 @@
+When(/^I configure the bugsnag endpoint$/) do
+  steps %Q{
+    When I set environment variable "BUGSNAG_ENDPOINT" to "http://#{current_ip}:#{MOCK_API_PORT}"
+  }
+end
+When(/^I navigate to the route "(.*)" on port "(\d*)"/) do |route, port|
+  steps %Q{
+    When I open the URL "http://localhost:#{port}#{route}"
+    And I wait for 1 second
+  }
+end
+Then(/^the request used payload v4 headers$/) do
+  steps %Q{
+    Then the "bugsnag-api-key" header is not null
+    And the "bugsnag-payload-version" header equals "4.0"
+    And the "bugsnag-sent-at" header is a timestamp
+  }
+end
+Then(/^the request contained the api key "(.*)"$/) do |api_key|
+  steps %Q{
+    Then the "bugsnag-api-key" header equals "#{api_key}"
+    And the payload field "apiKey" equals "#{api_key}"
+  }
+end
+Then(/^the request used the "(.*)" notifier$/) do |notifier_name|
+  bugsnag_regex = /^http(s?):\/\/www.bugsnag.com/
```

Comment:
This shouldn't be asserted in a generic step since most notifier libraries use the GitHub URL rather than the homepage.

Reply:
What should notifiers use here? Is it in the notifier spec? Alex can you make sure the notifier spec is clear once delisa has answered

Reply:
I've updated the spec, and will replace the regex step with a `not-null` step.